system/tests/test_completer.py

```diff
@@ -53,7 +53,7 @@ def test_completion_08(self):
         assert newline == 'git branch '
 
     def test_completion_09(self):
-        newline, possibilities = self.complete('$STASH_')
+        newline, possibilities = self.complete('$STASH_R')
```

Comment:
Is there a __$STASH_R__defined or was this supposed to be __$STASH_ROOT__?

Reply:
@cclauss We test wether `$STASH_R` will be completed to `$STASH_ROOT`. This PR introduced an environment variable called `$STASH_PY_VERSION`, so the old `$STASH_` completion test completed to `$STASH_PY_VERSION` instead of `$STASH_ROOT` and failed.